packages/rapidpro/src/Adaptor.js

```diff
@@ -49,30 +46,32 @@ export function execute(...operations) {
  * @returns {Operation}
  */
 export function addContact(params, callback) {
-  return state => {
+  return async state => {
     const [resolvedParams] = expandReferences(state, params);
 
     const { host, apiVersion, token } = state.configuration;
 
     const url = `${host}/api/${apiVersion || 'v2'}/contacts.json`;
 
-    const config = {
-      url,
-      data: resolvedParams,
-      headers: { Authorization: `Token ${token}` },
-    };
+    return await fetch(url, {
```

Comment:
The structure of this is wrong. You should never `return await` and you should never `await.then()`.

So this needs to be like:
```
const response = await fetch(...)
const result = await response.json()
// etc
return nextState
```